src/app/client/src/app/modules/workspace/components/update-batch/update-batch.component.ts

```diff
@@ -372,7 +374,8 @@ export class UpdateBatchComponent implements OnInit, OnDestroy, AfterViewInit {
       startDate: startDate,
       endDate: endDate || null,
       createdFor: this.userService.userProfile.organisationIds,
-      mentors: _.compact(mentors)
+      mentors: _.compact(mentors),
+      cert_templates: []
```

Comment:
This should not be empty always. If certificates already exist then this code is removing the cert_teamplates.

Reply:
but to remove the template anyway we need to call the template add api with empty values. i think it won't cause any issue,